tests/test_welltest_dpds.py

```diff
@@ -55,7 +55,13 @@ def test_main(tmpdir, mocker):
     welltest_dpds.main()
     assert Path("welltest_output.csv").exists()
     Path("welltest_output.csv").unlink()
-
+    # test --phase
+    mocker.patch(
+        "sys.argv", ["welltest_dpds", DATAFILEPATH, "55_33-1", "--phase", "WATER"]
+    )
+    welltest_dpds.main()
+    assert Path("welltest_output.csv").exists()
+    Path("welltest_output.csv").unlink()
```

Comment:
Is there something in the csv that we can assert on that proves that the WATER argument was processed, and processed correctly?

Reply:
unfortunately there is nothing in the output files that specifies for which phase it has been calculated. So the test is actually only checking  that water can be an argument and that it will generate a csv file. The test data used have the same start/end date for the build up whatever the phase.

Reply:
However we could check the superposition time calculation which is using the rate (thus will change depending on the phase)...